src/Concordium/Client/Export.hs

```diff
@@ -141,12 +171,21 @@ decodeMobileFormattedAccountExport
   :: BS.ByteString -- ^ JSON with encrypted accounts and identities,
```

Comment:
The JSON is not necessarily an encryption anymore

Reply:
Updated documentation.